packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -240,10 +225,12 @@ export async function init({
     flex?: string | null;
     customInit?: string | null;
 }): Promise<void> {
+    const urlParams = new URLSearchParams(window.location.search);
+    const container = sap?.ushell?.Container ?? sap.ui.require('sap/ushell/Container') as typeof sap.ushell.Container;
     // Register RTA if configured
-    sap.ushell.Container.attachRendererCreatedEvent(async function () {
-        if (flex) {
-            const lifecycleService = await sap.ushell.Container.getServiceAsync<AppLifeCycle>('AppLifeCycle');
+    if (flex) {
+        container.attachRendererCreatedEvent(async function () {
```

Comment:
@tobiasqueck Don't we need the `attachRendererCreatedEvent` also for the `resetAppState`? My understanding was that I can only use `container.getServiceAsync`, when Renderer is created.